CMP1903_A1_2324/Die.cs

```diff
@@ -8,16 +8,14 @@ namespace CMP1903_A1_2324
 {
     internal class Die
     {
-        /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
-         */
-
-        //Property
-
-
-        //Method
-
+        private static Random diceRoll = new Random();
+        public int CurrentRoll { get; private set; }
 
+        public int Roll()
+        {
+            CurrentRoll = diceRoll.Next(1, 7);
+            return CurrentRoll;
+        }
     }
+
 }
```

Comment:
Code follows C# style conventions. Functions well. Could comment on what each component of your code does.